src/github.com/getlantern/flashlight/settings/settings.go

```diff
@@ -32,51 +38,108 @@ type Settings struct {
 	AutoReport   bool
 	AutoLaunch   bool
 	ProxyAll     bool
+	InstanceID   string
+
+	sync.RWMutex
 }
 
-func Configure(cfg *config.Config, version, revisionDate string, buildDate string) {
+// Load loads the initial settings at startup, either from disk or using defaults.
+func Load(version, revisionDate, buildDate string) {
+	// Create default settings that may or may not be overridden from an existing file
+	// on disk.
+	settings = &Settings{
+		Version:      version,
```

Comment:
Good question. I guess they're only used in the UI correct? I don't see much harm in writing them to the file, but I could also probably add an easy comment/tag that would remove them. Is that what you're suggesting?